apps/alerting/notifications/pkg/apis/resource/routingtree/v0alpha1/routingtree_spec_gen.go

```diff
@@ -23,7 +23,7 @@ type MatcherType string
 // Route defines model for Route.
 // +k8s:openapi-gen=true
 type Route struct {
-	Continue          bool      `json:"continue,omitempty"`
+	Continue          bool      `json:"continue"`
```

Comment:
Yes and no. The app-sdk generator does not offer an option here. If I define 
```cue
continue?: bool
```
the generator creates a pointer field, e.g. `Continue *bool` which is a little bit confusing because it will always be null or true.